src/Web/Sfa.Das.Sas.ApplicationServices/Handlers/RemoveBasketHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Sfa.Das.Sas.ApplicationServices.Commands;
+using Sfa.Das.Sas.Shared.Basket.Interfaces;
+using Sfa.Das.Sas.Shared.Basket.Models;
+
+namespace Sfa.Das.Sas.ApplicationServices.Handlers
+{
+    
+    public class RemoveBasketHandler : IRequestHandler<RemoveBasketCommand, Guid>
+    {
+
+        private readonly IApprenticeshipFavouritesBasketStore _basketStore;
+
+        public RemoveBasketHandler(IApprenticeshipFavouritesBasketStore basketStore)
+        {
+            _basketStore = basketStore;
+        }
+
+        public async Task<Guid> Handle(RemoveBasketCommand request, CancellationToken cancellationToken)
+        {
+            var basket = await GetBasket(request);
```

Comment:
Do we need to check if it exists before removing? i think it should be safe to just call remove